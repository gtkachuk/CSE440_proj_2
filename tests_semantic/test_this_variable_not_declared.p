program testThisVariableNotDeclared;


class testThisVariableNotDeclared

BEGIN
   VAR
      someOtherVar : integer;


FUNCTION testThisVariableNotDeclared;
VAR someVar : integer;
BEGIN
   this.someVar := 7;
   someVar := 5;
   this.someVar := someVar;

   PRINT this.someVar
END

END
.
