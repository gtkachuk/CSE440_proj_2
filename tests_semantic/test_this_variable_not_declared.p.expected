ERROR:14:Variable "someVar" not declared
ERROR:16:Variable "someVar" not declared
ERROR:19:Variable "someVar" not declared
Errors detected. Exiting.
